app/[showStatus]/[showNameAndSeason]/contestants/page.tsx

```diff
@@ -0,0 +1,90 @@
+import { getCompetingEntityList, getTeamList, ITeam } from "../../../utils/wikiQuery";
+import { getWikipediaContestantData } from "../../../utils/wikiFetch";
+
+// This forces Next to only generate routes that exist in generateStaticParams, otherwise return a 404
+export const dynamicParams = false
+
+interface showProperties {
+  showNameAndSeason: string,
+  showStatus: string
+}
+
+// Creates routes for scoring
+export function generateStaticParams() {
+  
+    // Necessary Node modules to fetch data
+    const fs = require('fs');
+    const path = require('path');
+    
+    // Based on availability in leagueData
+    const pathToLeagueData = path.join(process.cwd(), 'app', 'leagueConfiguration');
+    const shows:Array<showProperties> = [];
+    fs.readdirSync(pathToLeagueData).map((file: string) => {
```

Comment:
Let's change the references to leagueData since this isn't using that!
```suggestion
    // Based on availability in leagueConfiguration
    const pathToLeagueConfiguration = path.join(process.cwd(), 'app', 'leagueConfiguration');
    const shows:Array<showProperties> = [];
    fs.readdirSync(pathToLeagueConfiguration).map((file: string) => {
```